src/ngrx/3-ngrx-init/ngrx-init.md

```diff
@@ -30,7 +38,7 @@ First, install [NgRx schematics](https://ngrx.io/guide/schematics/install#instal
 This command will install NgRx schematics and update `angular.json`:
 
 ```bash
-ng add @ngrx/schematics@14
+npx ng add @ngrx/schematics@14
```

Comment:
This might be an obvious question, but since which version is npx available?

Reply:
lol no idea xD but I've used it since Node 8/10 I think

Reply:
google: "npx comes bundled with npm version 5.2+ ", don't know which node versions pair with npm 5.2, but seems like an old feature